kubebuilder/quickstart: add instruction for enabling status subresource (#2897)
operator-framework/operator-sdk: website/content/en/docs/kubebuilder/quickstart.md

}
```

> Add the `+kubebuilder:subresource:status` [marker comment][status_marker] to enable the [status subresource][status_subresource] for the CRD so that the controller can update the CR status without changing the rest of the CR object:

Reviewer: Nit: wording, comment is implied by marker.
```suggestion
Add the `+kubebuilder:subresource:status` [marker][status_marker] to add a [status subresource][status_subresource] to the CRD manifest so that the controller can update CR statuses efficiently:
```

Author: Agree with the other nits but I'd still want to highlight exactly why we're enabling the status subresource rather than say `efficiently`. That behavior is not obvious and it's better to be verbose about it.
Part of the reason why I had this error in the first place.
